src/App.js

```diff
@@ -16,24 +16,26 @@ class App extends Component {
 			currentUser: '',
 			id: '',
 			foundMovie: '',
+			ratings: []
 			//maybe consider Nan lets do research
 		};
 	}
 
 	logInUser = user => {
-		console.log(window.location);
 		this.setState({ currentUser: user.user.name, id: user.user.id });
+		this.getUserRatings();
 		// this.setState(user.user) potential refactor later
 	};
 
 	logOutUser = () => {
 		this.setState({ currentUser: '', id: '' });
+		window.location.pathname = '/'
 	};
 
 	findMovieShowInfo = () => {
-		let movieID = window.location.pathname.split('/')
+		let movieID = parseInt(window.location.pathname.split('/')[2]);
 		if(!this.state.foundMovie || this.state.foundMovie.id !== movieID){
-		fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID[2]}`)
+		fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}`)
 			.then(response => {
 				if (response.ok) {
```

Comment:
So this is a really good question, because intitially we had an alert firing if the response wasn't ok, but if the response wasn't ok our catch was seemed to be running.